--- conf/console.default.ini.orig	2015-08-04 09:22:29.833001181 -0400
+++ conf/console.default.ini	2015-08-04 09:20:19.516001274 -0400
@@ -73,7 +73,7 @@
 session.keyini = /etc/eucaconsole/session-keys.ini
 session.httponly = true
 # Secure session implies SSL setup
-session.secure = false
+session.secure = true
 # Idle timeout (1800 sec = 30 min)
 session.timeout = 1800
 # Absolute timeout (43200 sec = 12 hours)
