--- conf/console.default.ini	2016-02-26 11:31:41.880004509 -0500
+++ conf/console.default.ini	2016-02-26 11:31:55.554004501 -0500
@@ -114,7 +114,7 @@
 workers = 4
 # if you use gunicorn outside of CentOS 6, you can set worker_class to "gevent"
 # on CentOS 7 RPMs we set this to "gevent" using a patch in the rpm spec
-worker_class = eucaconsole.gunicorn.ggevent.GeventWorker
+worker_class = gevent
 # set the tmp_upload_dir to change the directory where uploaded files will be stored. Must be writable by 'eucaconsole' user.
 #tmp_upload_dir = /var/tmp
 
